_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -877,18 +893,6 @@ entries:
             - "202204.0"
             - "202212.0"
 
-          - title: Shopping lists
-            nested:
-              - title: Managing shopping lists
-                url: /docs/marketplace/dev/glue-api-guides/shopping-lists/managing-shopping-lists.html
-                include_versions:
-                - "202108.0"
-                - "202204.0"
-                - "202212.0"
-
-              - title: Managing shopping list items
-                url: /docs/marketplace/dev/glue-api-guides/shopping-lists/managing-shopping-list-items.html
-
```

Comment:
Is there a good reason for deleting this?

Reply:
Yes, there were 2 sidebar entries called Shopping lists. Both linked to the same pages so the one I deleted is a duplicate.